packages/common-all/src/constants.ts

```diff
@@ -62,6 +62,10 @@ export enum ERROR_STATUS {
   UNKNOWN = "unknown",
 }
 
+export enum USER_MESSAGES {
```

Comment:
nice, ty - let's start consolidating to this centralized location for user strings.

Reply:
We might want to consider laying the groundwork in an extensible way  (to allow localization to happen in the future), but this is a step towards right direction.